test/typedoc.js

```diff
@@ -16,5 +16,13 @@ describe('TypeDoc', function() {
             Assert.notEqual(expanded.indexOf(Path.join(inputFiles, 'class.ts')), -1);
             Assert.equal(expanded.indexOf(inputFiles), -1);
         });
+        it('honors the exclude argument even on a fixed file list', function() {
+            var inputFiles = Path.join(__dirname, 'converter', 'class');
+            application.options.setValue('exclude', '**/class.ts');
+            var expanded = application.expandInputFiles([inputFiles]);
+
+            Assert.equal(expanded.indexOf(Path.join(inputFiles, 'class.ts')), -1);
+            Assert.equal(expanded.indexOf(inputFiles), -1);
+        });
```

Comment:
Could we add a few more test cases? There are quite a few GitHub issues regarding exclude so better testing will help us sort out what problems actually exist. For example, could you add a test with multiple excludes (see [#170 comment](https://github.com/TypeStrong/typedoc/issues/170#issuecomment-228983184))?